command/resource/client/client.go

```diff
@@ -918,6 +918,7 @@ func durToMsec(dur time.Duration) string {
 }
 
 // toHTTP converts the request to an HTTP request
+// request is a customer object that include use-case specific config
```

Comment:
Not sure this is the right place for this comment, maybe we could add it at the `request` definition site?